doc/ref_arch/openstack/chapters/chapter04.rst

```diff
@@ -1773,8 +1628,24 @@ may need to be made for certain Edge clouds depending upon their
 resource capabilities. It should be noted that none of these changes
 affect the definition of OpenStack flavors.
 
-Edge Cloud Deployment
-^^^^^^^^^^^^^^^^^^^^^
+The previous section listed the OpenStack services deployed on the controller nodes depepnding upon
+the control plane distribution. As specified earlier in this chapter, at least 3 controller nodes should be deployed 
+for HA.  Compute nodes may also exist at the sites where controller nodes are deployed.  
+
+Control plane services are not hosted at edge sites. Each edge site can be treated as its own OpenStack AZ. The 
```

Comment:
Just to double check. Is that clarified in RM or in this doc that the control plane services are deployed centrally to follow telecom requirements as opposed to being an OpenStack limitation?

Reply:
@ildikov RM doesn't deal with deployment and so this control plane distribution is only in the RA.